roles/openshift_logging/defaults/main.yml

```diff
@@ -69,13 +69,15 @@ openshift_logging_kibana_ops_key: ""
 openshift_logging_kibana_ops_ca: ""
 
 openshift_logging_fluentd_nodeselector: "{{ openshift_hosted_logging_fluentd_nodeselector_label | default('logging-infra-fluentd=true') | map_from_pairs }}"
-openshift_logging_fluentd_cpu_limit: 100m
-openshift_logging_fluentd_memory_limit: 512Mi
+openshift_logging_fluentd_cpu_limit: "{{ openshift_hosted_logging_fluentd_cpu_limit | default('100m') }}"
```

Comment:
Can you please also update the defaults for `roles/openshift_logging_fluentd/defaults/main.yml` ?